Sources/SmileID/Classes/Networking/Models/Services.swift

```diff
@@ -0,0 +1,153 @@
+import Foundation
+
+// MARK: - ServicesResponse
+
+public struct ServicesResponse: Codable {
+    public let bankCodes: [BankCode]
+    public let hostedWeb: HostedWeb
+
+    enum CodingKeys: String, CodingKey {
+        case bankCodes = "bank_codes"
+        case hostedWeb = "hosted_web"
+    }
+}
+
+// MARK: - BankCode
+
+public struct BankCode: Codable {
+    let name, code: String
+}
+
+// MARK: - HostedWeb
+
+public struct HostedWeb: Codable {
+    var basicKyc: CountryCodeToCountryInfo
+    var biometricKyc: CountryCodeToCountryInfo
+    var enhancedKyc: CountryCodeToCountryInfo
+    var docVerification: CountryCodeToCountryInfo
+    var enhancedKycSmartSelfie: CountryCodeToCountryInfo
+
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        basicKyc = try container.decode(CountryCodeToCountryInfo.self, forKey: .basicKyc)
```

Comment:
I'm slightly confused why we need to manually decode all these. Shouldn't declaring the CodingKeys take care of it? i.e. we are not doing this in other Models